python/cuml/manifold/umap.pyx

```diff
@@ -437,6 +431,24 @@ class UMAP(Base):
         elif isinstance(X, cupy.ndarray):
             return cupy.array(embedding)
 
+    @staticmethod
+    def find_ab_params(spread, min_dist):
+        """Fit a, b params for the differentiable curve used in lower
+        dimensional fuzzy simplicial complex construction. We want the
+        smooth curve (from a pre-defined family with simple gradient) that
+        best matches an offset exponential decay.
```

Comment:
I think we should note here that this function has been copied from UMAP-learn. Will help future developers, especially in the cases where updates are made on the umap-learn side.